### PR TITLE
Removed old and added two new examples for Application Autoscaling

### PR DESCRIPTION
**Pull request by bisdavid**
I added two new examples to the put-scaling-policy command in place of an older one.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by bisdavid** [comment]
I need help understanding why the Travis-CI checks failed please?

**Comment by dstufft** [comment]
It looks like somewhere in here you got non-ascii characters in the example file. You'll need to verify that the file is ascii only and doesn't contain any other strange character types or bytes.

File: awscli/examples/application-autoscaling/put-scaling-policy.rst [hunk 1]
```diff
@@ -1,33 +1,86 @@
...
+    aws application-autoscaling put-scaling-policy --service-namespace ecs \
+    --scalable-dimension ecs:service:DesiredCount \
+    --resource-id service/default/web-app \
+    --policy-name cpu75-target-tracking-scaling-policy --policy-type TargetTrackingScaling \
+    --target-tracking-scaling-policy-configuration file://config.json
+    
+This example assumes that you have a `config.json` file in the current directory with the following contents::
 
-  aws application-autoscaling put-scaling-policy --cli-input-json file://scale-out.json
+{
```
Comment (JordonPhillips) [inline awscli/examples/application-autoscaling/put-scaling-policy.rst]: This needs to be indented over

File: awscli/examples/application-autoscaling/put-scaling-policy.rst [hunk 2]
```diff
@@ -1,33 +1,86 @@
...
+  "TargetValue": 75.0,
+  "PredefinedMetricSpecification": {
+      "PredefinedMetricType": "ECSServiceAverageCPUUtilization"
+      },
+   "ScaleOutCooldown": 60,
+   "ScaleInCooldown": 60
+}
+    
+Output::
+{
```
Comment (JordonPhillips) [inline awscli/examples/application-autoscaling/put-scaling-policy.rst]: as does all of this

**Comment by bisdavid** [comment]
Was able to perform a local build of the CLI to verify that these were good. Please advise if otherwise.

**Pull request merged**